druid/styles/default/style.lua

```diff
@@ -84,7 +84,7 @@ M["scroll"] = {
 	INERT_SPEED = 30, -- koef. of inert speed
 	EXTRA_STRETCH_SIZE = 100, -- extra size in pixels outside of scroll (stretch effect)
 	POINTS_DEADZONE = 20, -- Speed to check points of interests in no_inertion mode
-	WHEEL_SCROLL_SPEED = 0, -- Amount of pixels to scroll by one wheel event (0 to disable)
+	WHEEL_SCROLL_SPEED = 40, -- Amount of pixels to scroll by one wheel event (0 to disable)
```

Comment:
Please don't change default style here. It's also non documented and better to disable mouse scroll by default (it's still not perfect)